reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -311,8 +423,8 @@ private synchronized ReactiveSocket select() {
             if (i2 >= i1) {
                 i2++;
             }
-            rsc1 = buffer.get(i1);
-            rsc2 = buffer.get(i2);
+            rsc1 = activeSockets.get(i1);
+            rsc2 = activeSockets.get(i2);
             if (rsc1.availability() > 0.0 && rsc2.availability() > 0.0)
```

Comment:
Same comment as the previous P2C code, should we select one with non `0.0` availability if the other is `0.0`